backend/src/email/middlewares/email-template.middleware.ts

```diff
@@ -35,7 +36,10 @@ const storeTemplate = async (
 ): Promise<Response | void> => {
   try {
     const { campaignId } = req.params
-    const { subject, body, reply_to: replyTo } = req.body
+    const { subject, body, reply_to: replyTo, from } = req.body
+
+    const fromAddress = from === 'default' ? config.get('mailFrom') : from
```

Comment:
@jeantanzj Unsure about my implementation of `from` as `default`. Let me know what you think!